chromium_src/chrome/browser/ui/views/profiles/profile_menu_view_base.cc

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+// Continue to use profile_background_container_ to set the background color in
+// the profile menu, rather than identiy_info_color_callback_
```

Comment:
Please include the "why" for these kinds of changes and do we have a planned follow-up for this?

Reply:
Updated comment with info from commit here 4b9aa78f2106938a386772ec5f71104d85012b1d and created https://github.com/brave/brave-browser/issues/39220 follow-up issue to remove or simplify this override.